### ISSUE TITLE
make `aria-label` configurable

### ISSUE THREAD

**Issue by jeroenbourgois** [dbushell/Pikaday#687]
Right now an `aria-label="Use the arrow keys to pick a date"` is set on the input element, but the copy cannot be changed with the config? Am I correct? If so, can I propose a PR to make this happen?

**Comment by rikkert** [comment]
Yes please.
